Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -456,14 +457,15 @@ public async Task Submit(SubmitSubmissionServiceModel model)
         await this.submissionsData.Add(newSubmission);
         await this.submissionsData.SaveChanges();
 
-        await this.submissionsForProcessingData.Add(newSubmission.Id);
+        submissionServiceModel = this.BuildSubmissionForProcessing(newSubmission, problem, submissionType);
+        await this.submissionsForProcessingData.Add(newSubmission.Id, submissionServiceModel.ToJson());
```

Comment:
And this as well: submissionServiceModel.ToJson()

Reply:
Same as above.